testing/src/main/java/io/grpc/internal/testing/AbstractTransportTest.java

```diff
@@ -1054,6 +1056,22 @@ public Void answer(InvocationOnMock invocation) throws Exception {
     assertEquals(status.getDescription(), statusCaptor.getValue().getDescription());
   }
 
+  @Test
+  public void clientStreamGetAttributes() throws Exception {
```

Comment:
As discussed offline. Move this and the modifications in `NettyTransportTest` to `NettyClientTransportTest`. It seems like it shouldn't be hard, and may even be easier than this current testing. Doing the move would prevent putting test-specific code in a shared location and does not disable the normal negotiator for `NettyTransportTest`.